beta-component-promotion/promotion-2022.01/beta-promotion-candidates.md

```diff
@@ -0,0 +1,19 @@
+# PatternFly Beta Component Promotion Candidates
+## For Release 2022.01
+
+### Back to top
+The Back to top component improves ease of navigation. It allows users to programmatically return to the top of the content.
+
+![back-to-top](./img/back-to-top.png)
+
+### Calendar month
+A calendar month component allows users to select and navigate between days, months and/or years. This component is usually used with the date picker component, to display date options and selections. 
+
+![calendar-month](./img/calendar-month.png)
+
+### Jump links
+Jump links provide a way to navigate through section headings on a page without scrolling. The highlighted active section changes as the user scrolls to the next section.
```

Comment:
```suggestion
A jump links component provides a way to navigate through section headings on a page without scrolling. The highlighted active section changes as the user scrolls to the next section.
```

Reply:
To make all descriptions consistent, I left some light edits so that they all follow the same structure: "A {{component name}} component"